OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/19.3.0/initsh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Since: January, 2022
+# Author: sanjay.singh@oracle.com,  paramdeep.saini@oracle.com
+# Description:
```

Comment:
There is no description (or a need for one, really). I would just remove this line.